public/app.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { I18nProvider } from '@osd/i18n/react';
+import { Route, RouteComponentProps, Switch } from 'react-router-dom';
+import { EuiPageSideBar, EuiSideNav, EuiPageTemplate } from '@elastic/eui';
+import { CoreStart } from '../../../src/core/public';
+import { Navigation, APP_PATH } from './utils';
+import { Overview, UseCases, Workflows } from './pages';
+import { CoreServicesConsumer } from './core_services';
+
+interface Props extends RouteComponentProps {}
+
+export const AiFlowDashboardsApp = (props: Props) => {
+  const sidebar = (
+    <EuiPageSideBar style={{ minWidth: 190 }} hidden={false}>
```

Comment:
From what I've seen typically yes, but it is also reasonable to include in-line for styling that is only used once, and for one component/element. Otherwise you would be adding too much boilerplate where a single css property would mean creating new css file, importing css file, and still having to reference the classname property for the component